EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
-QUEST_LV_0300_20160310_001368	Someone is definitely disturbing the coral pick ups.{nl}I don't know why, but he is so evil.
-QUEST_LV_0300_20160310_001369	Yes.{nl}I told you that the landlord is little strange.{nl}
-QUEST_LV_0300_20160310_001370	The lord lieutenant going out defeat the monsters would be quite efficient.{nl}Even if the offerings delivery gets delayed a bit.{nl}
-QUEST_LV_0300_20160310_001371	But, he didn't do it.{nl}
-QUEST_LV_0300_20160310_001372	If he knows defeating the monsters is useless..{nl}I think all the stories are getting tied.{nl}
-QUEST_LV_0300_20160310_001373	If what I am thinking is correct, the agent of the landlord will try to steal tributes.{nl}But, we don't have the decisive evidence.{nl}
-QUEST_LV_0300_20160310_001374	Okay.. Let's finish our guess here.  Shall we lure the suspect out?{nl}I want you to be responsible for the most important role.
-QUEST_LV_0300_20160310_001375	I've told the agent in advance.{nl}The odor was spreading badly at the pit so I've eliminated the source and they thanked me for that.{nl}
-QUEST_LV_0300_20160310_001376	In the meantime, please hide with the elder.{nl}Of course, the soldier who would catch the criminal should go together. He should act like an observer.{nl}
-QUEST_LV_0300_20160310_001377	Of course, we can't say the soldiers has participated..but..{nl}You don't have to worry since Basilius is reliable and a very close friend of mine.{nl}
-QUEST_LV_0300_20160310_001378	Anways, since the agent found out that he pulled out all the pockets..{nl}He will appear again and bury the pocket.{nl}
-QUEST_LV_0300_20160310_001379	Let's strike the place all at once then.{nl}In case something occurs, I will block the retreat route.
+QUEST_LV_0300_20160310_001331	I regret that I may have done something wrong due to hunger.{nl}If Dominikas came instead of you... I guess we won't be able to go back...
+QUEST_LV_0300_20160310_001332	I am worried about the abbot.{nl}He is old... I hope the goddesses take care of him...
+QUEST_LV_0300_20160310_001333	Ahah... No matter how much I think, there is no way.{nl}Now, everything may be ruined...{nl}
+QUEST_LV_0300_20160310_001334	I haven't seen you before.{nl}What has brought you this far?{nl}
+QUEST_LV_0300_20160310_001335	It doesn't matter.{nl}If there's nothing special, can you listen to this old man will you?
+QUEST_LV_0300_20160310_001336	This was... originally a sea...{nl}We were fishers who caught fishes in this sea.{nl}
+QUEST_LV_0300_20160310_001337	But, as you can see...{nl}The land suddenly soared upwards.{nl}
+QUEST_LV_0300_20160310_001338	At first, I was thinking whether I should quit fishing, but... it soon turned out alright.{nl}I've earned a lot of money by picking up corals. {nl}
+QUEST_LV_0300_20160310_001339	The monsters were without a doubt a bit scary, but the lord was there to help us.{nl}The lord sent her army and lieutenant in exchange for a bit of offerings.{nl}
```

Comment:
"were, without a doubt, a bit scary" 

"for a bit of offerings" -> "for an offering."